BGAnimations/ScreenSelectMusic overlay/PaneDisplay.lua

```diff
@@ -20,34 +20,20 @@ local GetSongAndSteps = function(player)
 end
 
 -- -----------------------------------------------------------------------
--- requires a profile (machine or player) as an argument
--- returns formatted strings for player tag (from ScreenNameEntry) and PercentScore
-
-local GetScoreAndName = function(profile, SongOrCourse, StepsOrTrail)
+local GetLocalScore = function(profile, SongOrCourse, StepsOrTrail)
```

Comment:
Can we add a function comment that the profile here can be a machine profile or a local profile (and probably usb profiles as well)